$NetBSD: patch-runtime_CMakeLists.txt,v 1.1 2015/08/10 23:19:00 jonthn Exp $

Awk scripts are not executable install them as simple files

--- runtime/CMakeLists.txt.orig	2015-08-05 23:39:13.000000000 +0200
+++ runtime/CMakeLists.txt	2015-08-05 23:40:04.000000000 +0200
@@ -66,7 +66,7 @@
 
 file(GLOB_RECURSE RUNTIME_PROGRAMS
   RELATIVE ${CMAKE_CURRENT_SOURCE_DIR}
-  *.awk *.sh *.bat)
+  *.sh *.bat)
 
 foreach(PROG ${RUNTIME_PROGRAMS})
   get_filename_component(BASEDIR ${PROG} PATH)
@@ -76,7 +76,7 @@
 
 file(GLOB_RECURSE RUNTIME_FILES
   RELATIVE ${CMAKE_CURRENT_SOURCE_DIR}
-  *.vim *.dict *.py *.ps)
+  *.vim *.dict *.py *.ps *.awk)
 list(APPEND RUNTIME_FILES macros/dvorak)
 
 foreach(F ${RUNTIME_FILES})
